index.html

```diff
@@ -2237,12 +2237,22 @@ <h2>Capability Invocation</h2>
         </dl>
 
         <p>
-An example of when this property is useful is when a <a>DID subject</a>
-chooses to invoke their capability to start a vehicle through the combined
-usage of a <a>verification method</a> and the <code>StartCar</code>
-capability. In this example, the vehicle would be the <em>verifier</em> and
-would need to verify that the <a>verification method</a> exists in the
-<code><a>capabilityInvocation</a></code> property.
+An example of when this property is useful is when a <a>DID subject</a> needs to
+access a protected HTTP API that requires authorization in order to use it. In
+order to authorize when using the HTTP API, the <a>DID subject</a>
+<em>invokes</em> a capability that is associated with a particular URL that is
+exposed via the HTTP API. The invocation of the capability is typically a
+digitally signed message that is placed into the HTTP Headers.
```

Comment:
-1 to "typically", +1 to describing an example.

```suggestion
exposed via the HTTP API. The invocation of the capability could be
expressed in a number of ways, e.g., as a digitally signed
message that is placed into the HTTP Headers.
```